### DISCUSSION TITLE
Downgraded pinned GitHub Action via Renovate PR

### DISCUSSION THREAD

**Discussion by TWiStErRob** [renovatebot/renovate#18397] [Request Help]
I found a strange PR: https://github.com/detekt/detekt/pull/5377 which downgrades the version upgraded in https://github.com/detekt/detekt/pull/5372.

Why would it downgrade? Investigation details in https://github.com/detekt/detekt/pull/5377#issuecomment-1283027507.

@rentziass do you have any idea why this might've happened? Was there something weird with v3 tag?

**Comment by rarkins** [comment]
<img width="403" alt="image" src="https://user-images.githubusercontent.com/6311784/196593556-bb59478d-deaf-4938-9a5b-363fbbbe853a.png">

For some reason it corresponds to the v3.0.2 tag, but we can't be sure that the tag was pointing to 14 days ago.

Renovate *is* wanting to revert it, but the PR is blocked because it was already merged previously:

<img width="714" alt="image" src="https://user-images.githubusercontent.com/6311784/196593745-5a8ad0db-6ef3-4cf1-8a63-af3f563c23b9.png">

**Reply by rarkins** [reply]
When I look back at the log of when the PR was done, you can see that Renovate queried the v3 tag URL: `https://api.github.com/repos/actions/checkout/git/refs/tags/v3`

And it was "seeing" the wrong SHA for quite some time until it started seeing the right one again.

Timestamps:
- `2022-10-05T08:46:07.522Z` - at this point it was correctly seeing the new SHA
- `2022-10-05T12:24:14.986Z` - this was when the "downgrade" occurred when it was the old one
- `2022-10-11T08:30:59.209Z` - this was the last time the error was present
- `2022-10-11T11:49:56.131Z` - this was when it started seeing the new SHA again

There was no change to the hosted Renovate app between the last two timestamps

**Reply by TWiStErRob** [reply]
Ooh, wow, thank you! Really nice answer 🫡.

Sorry, I didn't realize there's a dashboard. That clearly shows the action to do.

Are the logs public, or you have god rights because you're member of Renovate? If public how can I find them?

**Reply by rarkins** [reply]
People with write access to a repo can view logs for the last 3 days via app.renovatebot.com

As app admin I can still see logs for 30 days
